backend/openapi.yaml

```diff
@@ -328,7 +328,65 @@ paths:
         '525':
           description: SSL handshake failed
         '526':
-          description: Invalid SSL certificateÏ
+          description: Invalid SSL certificate
+
+  /campaign/{campaignId}/cancel:
```

Comment:
this should be `campaigns` now? dk whether you need to move the relative position of the doc